### DISCUSSION TITLE
수도권 전철 수인분당선 및 2호선에 서울지하철공사 1호선 초저항 지하철도색추가

### DISCUSSION THREAD

**Discussion by opentrain** [KoreanGRF/KoreanTrainSet#356] [건의(Suggestions)]
https://theqoo.net/square/1122994569
초저항 2호선전동차 운영다시 개통당시 신설동-종합운동장간 운영
![image](https://user-images.githubusercontent.com/81477455/181038198-c1113992-84e7-4d47-8cc5-7391d6a8afe8.png)

수인분당선(자석도색) 과거도색 해당이열차는 2017년부터 자석도색에서 지금의 분당선도색으로 바꿔져다고 들었습니다.
https://theqoo.net/square/501909883
![image](https://user-images.githubusercontent.com/81477455/181038818-e09a89b3-550d-41ef-9bd0-0e189a1f011c.png)

2호선:1호선 초저항도색넣으면 되지 않을까싶네요
수인분당선:VVVF 4호선 구도색 하고 3호선 GEC에 4호선 GEC에 반영된거처럼 하면 될지 않을까싶네요
또 2012~2017년까지 자석도색에서 변경되었다고 들었습니다.

**Comment by telk5093** [comment]
2호선, 수인분당선에 도색을 추가할 게 아니라 1호선의 해당도색 차량을 쓰면 됩니다. 필요하지 않다고 판단됩니다.
